src/www/ui/api/Models/Decider.php

```diff
@@ -151,22 +151,23 @@ public function setOjoDecider($ojoDecider)
    * Get decider as an array
    * @return array
    */
-  public function getArray($version = ApiVersion::V1)
+  public function getArray($version=ApiVersion::V2)
   {
-    if ($version == ApiVersion::V2) {
-      return [
-        "nomosMonk"  => $this->nomosMonk,
-        "bulkReused" => $this->bulkReused,
-        "newScanner" => $this->newScanner,
-        "ojoDecider" => $this->ojoDecider
-      ];
-    } else {
+    if($version == ApiVersion::V1){
       return [
         "nomos_monk"  => $this->nomosMonk,
         "bulk_reused" => $this->bulkReused,
         "new_scanner" => $this->newScanner,
         "ojo_decider" => $this->ojoDecider
       ];
+    }else{
+      return [
+        "nomosMonk"  => $this->nomosMonk,
+        "bulkReused" => $this->bulkReused,
+        "newScanner" => $this->newScanner,
+        "ojoDecider" => $this->ojoDecider
+      ];
     }
```

Comment:
Consider using V1 as the default pattern. I followed the same during upgrades. (`if` should check for `V2`, and `else` will handle all other cases).